client/pages/Dashboard/index.jsx

```diff
@@ -163,10 +165,18 @@ class Dashboard extends Component {
       }))
       : [];
 
+
     if (search.split('?token=')[1]) {
       localStorage.setItem('token', search.split('?token=')[1]);
     }
 
+    if (!localStorage.getItem('token')) {
+      const token = Cookies.get('jwt-token');
+      if (token) {
+        localStorage.setItem('token', token);
+      }
+    }
+
```

Comment:
Did you manually set the cookies to confirm this?

Reply:
Yeah...confirmed.